cli/project/files.go

```diff
@@ -5,36 +5,51 @@ import (
 	"os"
 	"path/filepath"
 
+	"github.com/apex/log"
 	"github.com/tarantool/cartridge-cli/cli/common"
 	"github.com/tarantool/cartridge-cli/cli/context"
 )
 
 const (
-	cartridgeLocalConf = ".cartridge.yml"
+	defaultHomeDir = "/home"
```

Comment:
Why not `$HOME`?

Reply:
See here - https://github.com/tarantool/cartridge-cli/pull/315/files#diff-660f4520ad6b4abeaabb003310e50b43R177

But you don't check access rights for this directory

Reply:
Fixed:

* `os.UserHomeDir()` is used now (BTW, it returns `${HOME}`);
* if getting home directory fails, cli returns an error.